### ISSUE TITLE
Publish current version to Atmosphere

### ISSUE THREAD

**Issue by smowden** [meteorclub/shareit#38]
The version on Atmosphere seems to be severyl outdated and I only found out by manually comparing the sources. 
Could you please publish the current version? As of now lovetostrikes package is actually more current on Atmosphere.

**Comment by queso** [comment]
Yeah, I am happy to publish it, I just didn't have a chance to test the recent pull requests.

**Comment by thomasspiesser** [comment]
any progress on this?
